src/seedu/addressbook/logic/Logic.java

```diff
@@ -85,7 +85,9 @@ public CommandResult execute(String userCommandText) throws Exception {
     private CommandResult execute(Command command) throws Exception {
```

Comment:
Do update the header comments for `execute`, since you are changing its expected behaviour.